package.json

```diff
@@ -114,7 +114,8 @@
     "db:validate": "nx run api:prisma:validate",
     "db:format": "nx run api:prisma:format",
     "db:reset": "nx run api:prisma:reset",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "sentry:sourcemaps": "sentry-cli sourcemaps inject --org programmingnotcoding --project node ./dist && sentry-cli sourcemaps upload --org programmingnotcoding --project node ./dist"
```

Comment:
I see some hardcoded properties in here!

Reply:
@HarshPatel5940 https://docs.sentry.io/platforms/node/sourcemaps/uploading/typescript/#2-configure-sentry-cli

Refer to this article for this.

Reply:
noted. removing it.

Reply:
i followed the article and pushed the changes accordingly.

Reply:
You also need to add that to `.env.example`